app/src/main/java/com/nononsenseapps/feeder/ui/compose/settings/OpenAISection.kt

```diff
@@ -150,6 +154,11 @@ fun OpenAISectionEdit(
     }
 
     var modelsMenuExpanded by remember { mutableStateOf(false) }
+    var timeoutString by remember { mutableStateOf(current.timeout) }
+    val isTimeoutInputValid =
+        timeoutString.isNotEmpty() &&
+            timeoutString.isDigitsOnly() &&
```

Comment:
Maybe add a `trim()` call and not only `isdigitsonly`. a pet peeve of mine is when apps and sites refuses to remove spaces from input fields.